src/win/winsock.c

```diff
@@ -131,6 +134,19 @@ void uv__winsock_init(void) {
     }
     closesocket(dummy);
   }
+
+  /* Try to get WSA function pointers */
+  dummy = socket(AF_INET, SOCK_STREAM, IPPROTO_TCP);
+  if (dummy != INVALID_SOCKET) {
+    const GUID wsaid_acceptex = WSAID_ACCEPTEX;
+    const GUID wsaid_connectex = WSAID_CONNECTEX;
+
+    /* If anyone failed, will return false when get them later. */
+    uv__get_extension_function(dummy, wsaid_acceptex, (void**)uv_wsa_acceptex);
```

Comment:
We can't work without them, can we? I'd abort() in that case.

Reply:
I see `tcp.c` will silently returns error, not crashing.